cachalot/tests/read.py

```diff
@@ -917,7 +954,7 @@ def test_now_annotate(self):
         """Check that queries with a Now() annotation are not cached #193"""
         qs = Test.objects.annotate(now=Now())
         self.assert_query_cached(qs, after=1)
-        
+
```

Comment:
weird space? I can use pre-commit later to remove this space though, so dw about it

Reply:
> weird space? I can use pre-commit later to remove this space though, so dw about it

Would it make sense to add the pro-commit configuration to the project itself?